src/encoded/tests/verifier.py

```diff
@@ -0,0 +1,113 @@
+from dcicutils.misc_utils import ignored
+from functools import wraps
+from snovault import TYPES
+from .test_create_mapping import test_create_mapping_correctly_maps_embeds
+#from .test_embedding import test_add_default_embeds, test_manual_embeds
+#from .test_schemas import compute_master_mixins, test_load_schema
+
+
+def verifier(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        print("running tests " + func.__name__)
+        try:
+            res = func(*args, **kwargs)
+        except Exception as e:
+            print("test failed with exception " + str(e.args))
+        else:
+            print("success")
+            return res
+    return wrapper
+
+
+@verifier
+def verify_get_from_es(item_uuid, indexer_testapp, registry):
+    ignored(registry)
+    # get from elasticsearch
+    es_item = indexer_testapp.get("/" + item_uuid + "/?datastore=elasticsearch").maybe_follow(status=200).json
+    item_type = es_item['@type'][0]
+    ensure_basic_data(es_item, item_type)
+    return es_item, item_type
+
+
+@verifier
+def verify_get_by_accession(es_item, item_type, indexer_testapp):
+    # get by accession
+    accession = es_item.get('accession')
+    if accession:  # some items don't have acessions
+        item_by_accession = indexer_testapp.get("/" + accession).follow(status=200).json
+        ensure_basic_data(item_by_accession, item_type)
+
+
+@verifier
+def verify_get_from_db(item_uuid, item_type, indexer_testapp):
+    # get from database
+    db_item = indexer_testapp.get("/" + item_uuid + "/?datastore=database").follow(status=200).json
+    ensure_basic_data(db_item, item_type)
+
+
+@verifier
+def verify_profile(item_type, indexer_testapp):
+    # is this something we actually know about?
+    profile = indexer_testapp.get("/profiles/" + item_type + ".json").json
+    assert profile
+    item_type_camel = profile['id'].strip('.json').split('/')[-1]
```

Comment:
could we have a separate (and tested) function that does this complicated dance so i know what it's achieving?